nemesis/arena_prep.py

```diff
@@ -1055,11 +1059,39 @@ def one_mp_ready():
         return False
     return True
 
+def tonberry_levels_battle():
+    screen.await_turn()
+    tidus_turn = False
+    while memory.main.battle_active():
+        if memory.main.turn_ready():
+            if screen.turn_tidus():
+                if tidus_turn == True:
+                    battle.main.tidus_flee()
+                elif memory.main.get_overdrive_battle(character=0):
+                    battle.overdrive.tidus()
```

Comment:
This can never happen. If it's his turn, hell flee

Reply:
you're not reading that correctly in context of the entire function, so I'ma just resolve this concern. Heavily tested and working as intended.

Reply:
The fact that I initially had this confusion likely means we ought to change `tidus_turn` to `tidus_should_flee`. Since that's actually what it's checking for.

Reply:
That's true. I'll make a note for the next round of development and update accordingly.